configserver/src/main/java/com/yahoo/vespa/config/server/modelfactory/ActivatedModelsBuilder.java

```diff
@@ -118,7 +123,7 @@ protected Application buildModelVersion(ModelFactory modelFactory,
                 wantedNodeVespaVersion);
         MetricUpdater applicationMetricUpdater = metrics.getOrCreateMetricUpdater(Metrics.createDimensions(applicationId));
         ServerCache serverCache = new ServerCache(configDefinitionRepo, zkClient.getUserConfigDefinitions());
-        return new Application(modelFactory.createModel(modelContext),
+        return new Application(withDeferredConfigForRestartingClusters(modelFactory.createModel(modelContext)),
```

Comment:
As far as I could see, this builder is the only one that is used for models that end up being used for serving config. Please correct me if I'm wrong, and the prepared-models-builder also does that.

Reply:
Yes, that's right models built by this will serve config